packages/bpk-svgs/README.md

```diff
@@ -29,7 +31,8 @@ This package exposes Backpack SVGs in various formats:
   bpk-svgs/dist/scss/_spinners.scss
   ```
 
-- An icon font
+- Icon fonts
   ```
-  bpk-svgs/dist/font/BpkIcon.ttf
+  bpk-svgs/dist/font/sm/BpkIcon.ttf
```

Comment:
I think this will break the app build process, If I'm not mistaken there is a phase where they copy the font files the RN assets folder

Reply:
Yeah that's exactly why it's a breaking change. When we upgrade the dependency we'll need to change the location it uses